CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
-- Fix: libmicrohttpd 0.9.48 included in contextBroker as static lib (previous Orion versions used 0.99.22 as dynamic library) (Issue #1675)
-- Fix: error level traces ignoring -logLevel NONE
-- Fix: wrong over-logging at error level updating attributes having metadata without type 
+-Fix: error level traces ignoring -logLevel NONE
+-Fix: wrong over-logging at error level updating attributes having metadata without type 
+-Fix: libmicrohttpd 0.9.48 included in contextBroker as static lib (previous Orion versions used 0.99.22 as dynamic library) (Issue #1675)
+-Fix: list of attribute names in URI param 'type' (Issue #1749)
```

Comment:
Looking to the last CNR version before the last cleanup (https://github.com/telefonicaid/fiware-orion/blob/a128b4c715c7e7d1881cc5614b13390f2dd6871e/CHANGES_NEXT_RELEASE) the "style" is to put one whitespace between "-" and "Fix".

(I have the feeling that each time CNR is flused each time we open release, we "flip" to another style variant :))